src/handler/getPersonById/index.ts

```diff
@@ -0,0 +1,19 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+
+import { getPersonById } from '../../pseudoDB/index';
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  const id = event.pathParameters && event.pathParameters.id ? parseInt(event.pathParameters.id, 10) : 0;
```

Comment:
What happens if this parameter is not present? Maybe you should thread it better, return a specific error etc

Reply:
If the request does not have the `id` it will use the `/person` route that returns all people. If he has an `id` that is not a numerical or from a person who does not exist returns 404. Should I treat if `id` is not a number?

Reply:
If it's not a number and you expect a number the best would be to show an error, so you make the api interface more strict.

Reply:
Done.